### PR TITLE
Added GitHub CLI option to Contributing section

### PR DESCRIPTION
**Pull request by ai-fast-track**
Updated the Contributing section, in docs, to include GitHub CLI option. Please fill up the TODO sections for the regular git option

### CODE REVIEW

**Comment by oke-aditya** [comment]
For regular git. Just update the repo after PR is opened. Add it commit it. 

Push to GitHub. GitHub will automatically update the PR.

**Comment by oke-aditya** [comment]
To confirm pr was created. Come to this repo.  Check in pr section.  simple.

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -62,19 +62,74 @@ Creating PR
...
 
-4. Create a pull request
+Step 5: Submitting a Pull Request
+=================================
 
-Upload your local branch to your remote GitHub repo (github.com/username/mantisshrimp)
+A. Method 1: Using GitHub CLI
+-----------------------------  
+
+Preliminary step (done only once): Install gh by following the instructions here -> https://cli.github.com/manual/installation
```
Comment (lgvaz) [inline docs/source/contributing.rst:71]:
```suggestion
Preliminary step (done only once): Install gh by following the instructions `here <https://cli.github.com/manual/installation>`_ .
```

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -62,19 +62,74 @@ Creating PR
...
+If you want to change your code after a PR has been created, you can do it by sending more commits to the same remote branch. 
+For example:
+
+.. code:: bash
+
+    git commit -m "updated the feature"
+    git push origin <enter-branch-name-same-as-before>
+
+It will automatically show up in the PR on the github page.
+If these are small changes they can be squashed together at the merge time and appear as a single commit in the repository.
```
Comment (lgvaz) [inline docs/source/contributing.rst:109]:
```suggestion
If these are small changes they can be squashed together by the reviewer at the merge time and appear as a single commit in the repository.
```

File: docs/source/contributing.rst [hunk 3]
```diff
@@ -62,19 +62,74 @@ Creating PR
...
+
+.. code:: bash
+   
     git push
 
 After the push completes, a message may display a URL to automatically submit a pull request to the upstream repo. 
 If not, go to the mantisshrimp main repo and GitHub will prompt you to create a pull request.
 
-5. Review
+2. Confirm PR was created : TODO
```
Comment (lgvaz) [inline docs/source/contributing.rst:125]:
```suggestion
2. Confirm PR was created:  

Ensure your pr is listed `here <https://github.com/lgvaz/mantisshrimp/pulls>`_
```

File: docs/source/contributing.rst [hunk 4]
```diff
@@ -62,19 +62,74 @@ Creating PR
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+3. Updating a PR : TODO
```
Comment (lgvaz) [inline docs/source/contributing.rst:128]:
```suggestion
3. Updating a PR:  

Same as before, normally push changes to your branch and the PR will get automatically updated.

.. code:: bash

    git commit -m "updated the feature"
    git push origin <enter-branch-name-same-as-before>
 
```

**Pull request merged**